client/asset/btc/btc_test.go

```diff
@@ -246,6 +246,10 @@ type tRawRequester struct {
 func (c *tRawRequester) RawRequest(_ context.Context, method string, params []json.RawMessage) (json.RawMessage, error) {
 	switch method {
 	// TODO: handle methodGetBlockHash and add actual tests to cover it.
+	case methodGetNetworkInfo:
+		return json.Marshal(&btcjson.GetNetworkInfoResult{
+			Connections: 1,
```

Comment:
If we're good with the general approach in this PR of reporting peer count changes in wallet state (and using a zero peer count to also block new orders), I'll update the tests to verify proper behavior when conns/peers goes to 0. Namely, force sync status to false.  The core test is already updated to ensure orders are blocked.